docs/scripts.rst

```diff
@@ -353,8 +358,15 @@ Full control over column names and data types
 
 By default the Retriever automatically detects both column names and data types, but you can also exercise complete
 control over the structure of the resulting database by adding column names and types.
+
+It is recommended to describe the schema of the table while creating JSON file. This enables processing of the data faster since column detection increases the processing time.
+
 These values are stored in the ``fields`` array of the ``schema`` dict of the JSON script.
 
+The ``fields`` enables full control of the columns, which includes, renaming columns, skipping unwanted columns, mentioning primary key and combining columns.
```

Comment:
" The ''fields'' **value** enables full control of the columns, which includes remaining columns, skipping unwanted columns, creating primary keys, and combining columns.